lib/tags/group-creator.tag

```diff
@@ -128,8 +128,10 @@
                         if(response.success) {
                             self.refs.title.classList.remove('graphjs-error');
                             self.refs.title.classList.add('graphjs-success');
+                            self.refs.title.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
                             self.refs.description.classList.remove('graphjs-error');
                             self.refs.description.classList.add('graphjs-success');
+                            self.refs.description.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
```

Comment:
Please remove this change. self.refs.description refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above